controller-api/src/main/java/com/yahoo/vespa/hosted/controller/api/integration/vcmr/VcmrReport.java

```diff
@@ -96,15 +96,18 @@ public String toString() {
     public static class Vcmr {
```

Comment:
This is exactly the kind of class that should become a record, even if IntelliJ doesn't want to suggest it. Or maybe there's some object-mapper related reason it can't be?

```java
    public record Vcmr (@JsonProperty("id") String id,
                        @JsonProperty("plannedStartTime") ZonedDateTime plannedStartTime,
                        @JsonProperty("plannedEndTime") ZonedDateTime plannedEndTime,
                        @JsonProperty("status") String status ) {}
```

Reply:
(IntelliJ doesn't suggest it because the constructor is not public and the instance variables are not final. But they can be.)

Reply:
>Or maybe there's some object-mapper related reason it can't be?

Nope, no reason